build/run-backup.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+set -e
+
+request_data() {
+	case "${STORAGE_TYPE}" in
+		"s3")
+			cat <<-EOF
+				{
+				    "destination": "${BACKUP_DEST}",
+				    "storage": {
+				        "type": "${STORAGE_TYPE}",
+				        "verifyTLS": ${VERIFY_TLS},
+				        "s3": {
+				            "bucket": "${S3_BUCKET}",
+				            "endpointUrl": "${AWS_ENDPOINT}",
+				            "accessKey": "${AWS_ACCESS_KEY_ID}",
+				            "secretKey": "${AWS_SECRET_ACCESS_KEY}",
+				            "region": "${AWS_DEFAULT_REGION}",
+				            "storageClass": "${S3_STORAGE_CLASS}"
+				        }
+				    }
+				}
+			EOF
+			;;
+		"gcs")
+			cat <<-EOF
+				{
+				    "destination": "${BACKUP_DEST}",
+				    "storage": {
+				        "type": "${STORAGE_TYPE}",
+				        "verifyTLS": ${VERIFY_TLS},
+				        "gcs": {
+				            "bucket": "${GCS_BUCKET}",
+				            "endpointUrl": "${GCS_ENDPOINT}",
+				            "accessKey": "${ACCESS_KEY_ID}",
```

Comment:
we need to ask @spron-in  about it , because in all our operators it is possible to  use ` AWS_ACCESS_KEY_ID` for gcp